public/js/common.js

```diff
@@ -6,12 +6,18 @@ const LRR = {};
 /**
  * Quick 'n dirty HTML encoding function.
  * @param {*} r The HTML to encode
+ * @param {boolean} isTimestamp Option to convert timestamp to date string
  * @returns Encoded string
  */
-LRR.encodeHTML = function (r) {
+LRR.encodeHTML = function (r, isTimestamp = false) {
```

Comment:
The timestamp conversion should probably go in a different function; encodeHTML encodes HTML, not various stuff on top of it. 😅